cpp/include/cudf/scalar/scalar_factories.hpp

```diff
@@ -156,5 +156,17 @@ std::unique_ptr<scalar> make_fixed_point_scalar(
   return std::make_unique<scalar_type_t<T>>(value, scale, true, stream, mr);
 }
 
+/**
+ * @brief Construct scalar using the given value of fixed_point type
```

Comment:
Sorry to jump in late here.

```suggestion
 * @brief Construct scalar using the given column of elements
```

Reply:
Better late than never ;)